src/handler_loader/emplacer.rs

```diff
@@ -91,11 +90,8 @@ impl Emplacer {
         self.emplace_bindle(&reader, id).await
     }
 
-    async fn emplace_remote_bindle(self, bindle_base_url: &Url, id: &bindle::Id) -> anyhow::Result<EmplacedHandlerConfiguration> {
-        let token = bindle::client::tokens::NoToken::default();
-        let client = bindle::client::Client::new(bindle_base_url.as_str(), token)?;
-
-        self.emplace_bindle(&client, id).await
+    async fn emplace_remote_bindle(self, id: &bindle::Id, bindle_connection_info: crate::bindle_util::BindleConnectionInfo) -> anyhow::Result<EmplacedHandlerConfiguration> {
```

Comment:
Consider reverting arg order to retain consistency with `emplace_standalone_bindle` 
```suggestion
    async fn emplace_remote_bindle(self, bindle_connection_info: crate::bindle_util::BindleConnectionInfo, id: &bindle::Id) -> anyhow::Result<EmplacedHandlerConfiguration> {
```